<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>964</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TRADING PLUS 2015</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/icons/icons/Dollar.png</normaloff>:/icons/icons/Dollar.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>871</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>700</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDevises">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>700</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>711</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu_Affiche">
    <property name="title">
     <string>&amp;Affichage</string>
    </property>
    <addaction name="actionCours_devises"/>
    <addaction name="actionGraphique"/>
    <addaction name="actionSimulation"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menu_Affiche"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuitter"/>
   <addaction name="actionCours_devises"/>
   <addaction name="actionGraphique"/>
   <addaction name="actionSimulation"/>
   <addaction name="action_Rafraichir"/>
   <addaction name="actionOptions"/>
   <addaction name="actionCalendrier"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/Exit.png</normaloff>:/icons/icons/Exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="toolTip">
    <string>Fermer l'application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCours_devises">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/oeil.png</normaloff>:/icons/icons/oeil.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cours devises</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionGraphique">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/3d bar chart.png</normaloff>
     <normalon>:/images/icons/3d bar chart.png</normalon>:/icons/icons/3d bar chart.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Graphique</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/Calculator.png</normaloff>
     <normalon>:/images/icons/Calculator.png</normalon>:/icons/icons/Calculator.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Simulation</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/Wrench.png</normaloff>:/icons/icons/Wrench.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/About.png</normaloff>
     <normalon>:/images/icons/About.png</normalon>:/icons/icons/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="action_Rafraichir">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/Sync.png</normaloff>:/icons/icons/Sync.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rafraichir</string>
   </property>
   <property name="toolTip">
    <string>Recharge la page</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionCalendrier">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/icons/Calendar.png</normaloff>:/icons/icons/Calendar.png</iconset>
   </property>
   <property name="text">
    <string>Calendrier</string>
   </property>
   <property name="toolTip">
    <string>Affichage par p√©riode entre 2 dates</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
